packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/network-form.template.html

```diff
@@ -66,17 +66,44 @@
                 data-model="$ctrl.subnet"
                 data-items="$ctrl.subnetsByRegion"
                 data-match="displayedLabel"
-                data-on-change="$ctrl.onSubnetChanged(modelValue)"
             >
             </oui-select>
         </oui-field>
     </div>
 
     <oui-message
         data-ng-if="!$ctrl.isLoadingSubnets && $ctrl.subnetError"
-        data-type="{{ $ctrl.subnetError.type }}"
+        data-type="error"
     >
-        <span data-ng-bind="$ctrl.subnetError.message"></span>
+        <span
+            data-translate="kubernetes_network_form_subnet_error_default"
+            data-translate-values="{ error: $ctrl.subnetError }"
+        ></span>
+    </oui-message>
+
+    <oui-message data-ng-if="$ctrl.shouldWarnSubnet" data-type="warning">
+        <p
+            data-translate="kubernetes_network_form_subnet_error_no_gateway_ip_p1"
+        ></p>
+        <p>
+            <span
+                data-translate="kubernetes_network_form_subnet_error_no_gateway_ip_p2"
+            ></span
+            >,
```

Comment:
**nitpick:** `comma` can be part of `kubernetes_network_form_subnet_error_no_gateway_ip_p2`